pyproject.toml

```diff
@@ -27,7 +26,7 @@ Documentation = "https://python-plexapi.readthedocs.io"
 version = {attr = "plexapi.const.__version__"}
 
 [build-system]
-requires = ["setuptools", "wheel"]
```

Comment:
Setuptools uses a custom version of `wheel` so the requirement here isn't used anymore.